docs/upgrade-5.0-6.0.md

```diff
@@ -0,0 +1,25 @@
+# Upgrade `5.0` to `6.0`
+
+Stop `temboard` service:
+
+``` shell
+sudo systemctl stop temboard
+```
+
+New package installation:
+
+``` shell
+sudo yum install temboard
+```
+
+Stamp `repository` database schema to latest version:
+
+``` shell
+$ temboard-migratedb stamp
```

Comment:
Would it be possible to run this command in a post-install script in RPM package?

Reply:
Non. Une installation d'un paquet peut avoir plusieurs instanciations. C'est le rôle d'un Ansible ou autre d'orchestrer la mise-à-jour. C'est la philosophie RH en général.